cli/migrate.go

```diff
@@ -283,42 +321,31 @@ func publishEventWithOptions(ctx context_pkg.Context, envName string, modifiedSc
 		if err := env.HandleEvent(eventName, eventContext); err != nil {
 			log.Fatalf("Failed to handle event '%s': %s", eventName, err)
 		}
+
+		if err := migration.UnmarkSchema(s.ID); err != nil {
+			log.Fatalf("Failed to remove '%s' from pending post migration schemas: %s", s.ID, err)
```

Comment:
I wonder if this fail can somehow affect which events will be executed after retry

Reply:
We'll trigger the post-migration event handling again. Handlers should be written in such a way to allow multiple runs already, nothing changes here.